cluster/definition.go

```diff
@@ -103,10 +92,10 @@ type Definition struct {
 	Threshold int `json:"threshold" ssz:"uint64" config_hash:"5" definition_hash:"5"`
 
 	// FeeRecipientAddress 20 byte Ethereum address.
-	FeeRecipientAddress []byte `json:"fee_recipient_address,0xhex" ssz:"Bytes20" config_hash:"6" definition_hash:"6"`
+	FeeRecipientAddress EthAddr `json:"fee_recipient_address,0xhex" ssz:"Bytes20" config_hash:"6" definition_hash:"6"`
```

Comment:
we cannot use bytes for internal representation of addresses since that looses checksummed information, so store as string rather.